app/templates/main/index.html

```diff
@@ -4,21 +4,122 @@
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAhkKJDnPxq0_xvF6pLAFcKfVAL6GbARBs&libraries=places&callback=initMap"
             async defer>
     </script>
+
+    <script type="text/javascript">
+      function showhide(id) {
+        var e = document.getElementById(id);
```

Comment:
if you're trying to hide something you can use jQuery like so:
``` 
$(#id).show()
$(#id).hide()
```
and also use jQuery to set the inside text by 
```
$(#id).text("set text to this")
```

Let's also move this script block to after the jinja elements, right before the `{% end block %}`